lib/taskana-core/src/main/resources/sql/taskana-schema.sql

```diff
@@ -57,6 +57,7 @@ CREATE TABLE WORKBASKET(
     ORG_LEVEL_2 VARCHAR(255) NULL,
     ORG_LEVEL_3 VARCHAR(255) NULL,
     ORG_LEVEL_4 VARCHAR(255) NULL,
+    MARKED_FOR_DELETION SMALLINT NOT NULL,
```

Comment:
Please increase schema version to 1.0.4 in all files and in the TaskanaEngine (as discussed yesterday)
We also need a migration script from 1.0.2 to 1.0.4

Reply:
I change the version in TaskanaEngineConfiguration, taskana-schema.sql, taskana-schema-db2.sql and taskana-schema-postgres.sql. Also, I create a new script for postgre (taskana_schema_update_1.0.2_to_1.0.4_postgre.sql) and a script for the other databases (taskana_schema_update_1.0.2_to_1.0.4.sql)